src/AspNet.Security.OAuth.Trakt/TraktAuthenticationOptions.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Microsoft.AspNetCore.Http;
+using static AspNet.Security.OAuth.Trakt.TraktAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.Trakt
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="TraktAuthenticationHandler"/>.
+    /// </summary>
+    public class TraktAuthenticationOptions : OAuthOptions
+    {
+        public TraktAuthenticationOptions()
+        {
+            ClaimsIssuer = TraktAuthenticationDefaults.Issuer;
+
+            CallbackPath = new PathString(TraktAuthenticationDefaults.CallbackPath);
+
+            AuthorizationEndpoint = TraktAuthenticationDefaults.AuthorizationEndpoint;
+            TokenEndpoint = TraktAuthenticationDefaults.TokenEndpoint;
+            UserInformationEndpoint = TraktAuthenticationDefaults.UserInformationEndpoint;
+
+            ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "username");
+            ClaimActions.MapJsonKey(ClaimTypes.Name, "username");
```

Comment:
Is this actually the user's name? For example in GitHub my username is `martincostello` but my name is _Martin Costello_.

Reply:
You cant provide your first and lastname at trakt

Reply:
But is it their _name_? If it's a user name, then it's not a value intended by the `Name` claim.